src/core/Button/Button.tsx

```diff
@@ -82,11 +92,12 @@ class BaseButton extends Component<ButtonProps> {
 
     return (
       <>
-        <HtmlButton
+        <HtmlButtonWithRef
```

Comment:
Do we have a use case for HtmlButton without the ref still? And if we do, could we just use Omit<HtmlButtonWithRefProps, 'ref'>? This way there would be no need for separate variants.